AppLovinAdapter/src/main/java/com/chartboost/mediation/applovinadapter/AppLovinAdapter.kt

```diff
@@ -668,6 +676,22 @@ class AppLovinAdapter : PartnerAdapter {
                         fullyWatched: Boolean
                     ) {
                         // TODO: HB-4119: Need to check if we need to add logic here for rewards.
+                        // Only stop the reward if the user was explicity deny listed by AppLovin
+                        if (isUserVerified == false) {
+                            PartnerLogController.log(
+                                CUSTOM,
+                                "Unable to reward due to user being denylisted by AppLovin."
+                            )
+                            return
+                        }
```

Comment:
We don't need to check against `percentViewed` and `fullyWatched` as well?

Reply:
Good point. I guess I read it as `videoPlaybackCompleted` or something. Thanks

Reply:
I'm just going to check fully watched. According to their docs, that's the one that will be used: https://support.applovin.com/hc/en-us/articles/4403802571661-When-is-the-best-time-to-reward-the-user-